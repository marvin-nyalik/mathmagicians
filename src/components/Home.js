import HomeCSS from './Home.module.css';

const home = () => (
  <div className={HomeCSS.cover}>
    <div className={HomeCSS.home}>
      <h2 className={HomeCSS.headline}>Welcome to Math-Magicians</h2>
      <div className={HomeCSS.description}>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto,
          ducimus fugit iste ea vel placeat nostrum magni incidunt molestias
          voluptates impedit corporis dignissimos eum facere nulla et! Eaque,
          saepe eveniet?
        </p>
        <p>
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Fugiat
          ducimus reprehenderit repellat nulla fugit ea a recusandae aspernatur
          quibusdam molestiae consequuntur voluptate esse ex quasi velit, illo
          tenetur ad dolor accusamus nam necessitatibus commodi. Fugiat magnam
          tempore itaque ullam sed.
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa autem
          magni deserunt architecto magnam praesentium.
        </p>
      </div>
    </div>
  </div>
);

export default home;
