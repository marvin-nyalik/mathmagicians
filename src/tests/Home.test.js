import { render } from '@testing-library/react';
import Home from '../components/Home';

describe('Test Home component', () => {
  it('Matches the snapshot', () => {
    const { container } = render(<Home />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
