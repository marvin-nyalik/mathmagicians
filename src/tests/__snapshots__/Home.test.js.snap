// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home component Matches the snapshot 1`] = `
<div
  class="cover"
>
  <div
    class="home"
  >
    <h2
      class="headline"
    >
      Welcome to Math-Magicians
    </h2>
    <div
      class="description"
    >
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto, ducimus fugit iste ea vel placeat nostrum magni incidunt molestias voluptates impedit corporis dignissimos eum facere nulla et! Eaque, saepe eveniet?
      </p>
      <p>
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Fugiat ducimus reprehenderit repellat nulla fugit ea a recusandae aspernatur quibusdam molestiae consequuntur voluptate esse ex quasi velit, illo tenetur ad dolor accusamus nam necessitatibus commodi. Fugiat magnam tempore itaque ullam sed.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa autem magni deserunt architecto magnam praesentium.
      </p>
    </div>
  </div>
</div>
`;
